### PR TITLE
feat: Replace exception with more user friendly message [DHIS2-9444]

### PR DESCRIPTION
**Pull request by zubaira**

### CODE REVIEW

**Review by enricocolasante — APPROVED** [review]
Ok for me, let's wait for Jim approval though, to check if this is not changing anything in Predictors

**Review by jimgrace — APPROVED** [review]
I like it! Very good change. Thanks!

**Pull request merged**
